app/src/main/java/com/example/dessertrelease/ui/DessertReleaseScreen.kt

```diff
@@ -108,12 +108,17 @@ private fun DessertReleaseScreen(
 }
```

Comment:
You should always set the inner padding to the content padding for lists. Please do the same for Grid layout also.
 ```
            modifier = Modifier
                    .padding(
                        vertical = dimensionResource(R.dimen.padding_medium),
                        horizontal = dimensionResource(R.dimen.padding_medium)
                    )
                    .fillMaxWidth(),
                contentPadding = innerPadding
            )
```